src/Domain.jl

```diff
@@ -18,16 +18,17 @@ mutable struct EnvLayer{S<:AbstractString,TF}
 end
 
 """
-    load_domain(path::String)
+    load_domain(path::String; coral_spec_path::String="")
 
 Load ADRIA domain specification from data package.
 No SSP/RCP data is preset.
 
 # Arguments
 - `path` : location of data package.
+- `coral_spec_path` : location of JSON with coral ecological parameters if not using default functional groups and size classes.
 """
-function load_domain(path::String)::Domain
-    return load_domain(path, "")
+function load_domain(path::String; coral_spec_path::String="")::Domain
+    return load_domain(path, ""; coral_spec_path=coral_spec_path)
```

Comment:
Sorry, I don't see how this works.

The Domain datasets themselves hold data relevant to initial coral cover, susceptibility to cyclones and other info.